chromadb/telemetry/posthog.py

```diff
@@ -21,10 +22,31 @@ def __init__(self, system: System):
         posthog_logger = logging.getLogger("posthog")
         # Silence posthog's logging
         posthog_logger.disabled = True
+
+        self.batched_event: Optional[TelemetryEvent] = None
```

Comment:
Is the assumption that we are OK with only one batched_event in flight at a time for now since only adds are batched? If we want to support batching other types concurrently (say interleaved adds/updates) we will have to change this a bit right? Is that worth just doing now

Reply:
That was the assumption but you're right, worth doing now. Done.